package.json

```
@@ -63,7 +63,7 @@
   "engines": {
     "node": "^18.18.0 || ^20.8.0 || ^22.0.0"
   },
-  "packageManager": "pnpm@9.2.0+sha512.98a80fd11c2e7096747762304106432b3ddc67dcf54b5a8c01c93f68a2cd5e05e6821849522a06fb76284d41a2660d5e334f2ee3bbf29183bf2e739b1dafa771",
+  "packageManager": "pnpm@10.5.0",
```

https://github.com/nodejs/corepack/blob/main/README.md#when-authoring-packages
> The hash is optional but strongly recommended as a security practice.

```suggestion
  "packageManager": "pnpm@10.5.0+sha512.11106a5916c7406fe4b8cb8e3067974b8728f47308a4f5ac5e850304afa6f57e2847d7950dfe78877d8d36bfb401d381c4215db3a4c3547ffa63c14333a6fa51",
```